tests/helix/send-to-helix-inner.proj

```diff
@@ -155,6 +156,8 @@
     <HelixPreCommand Include="$(_EnvVarSetKeyword) DCP_DIAGNOSTICS_LOG_LEVEL=debug" />
     <HelixPreCommand Include="$(_EnvVarSetKeyword) DCP_DIAGNOSTICS_LOG_FOLDER=$(_HelixLogsPath)" />
     <HelixPreCommand Include="$(_EnvVarSetKeyword) DCP_PRESERVE_EXECUTABLE_LOGS=1" />
+    <HelixPreCommand Condition="'$(OS)' == 'Windows_NT' and '$(_DockerCliPath)' != ''" Include="set PATH=$(_DockerCliPath)%3B%PATH%" />
+    <HelixPreCommand Condition="'$(OS)' != 'Windows_NT' and '$(_DockerCliPath)' != ''" Include="export PATH=$(_DockerCliPath):$PATH" />
```

Comment:
This will get set on helix where we don't have artifacts. Instead you would have the correlation payload expanded.

```suggestion
    <HelixPreCommand Condition="'$(OS)' == 'Windows_NT' and '$(_DockerCliPath)' != ''" Include="set PATH=%HELIX_CORRELATION_PAYLOAD%\docker-cli%3B%PATH%" />
    <HelixPreCommand Condition="'$(OS)' != 'Windows_NT' and '$(_DockerCliPath)' != ''" Include="export PATH=$HELIX_CORRELATION_PAYLOAD/docker-cli:$PATH" />
```

Reply:
Updated the precommands